### PR TITLE
release-check-list: Update release checklist

### PR DESCRIPTION
**Pull request by stevenhorsman**
- Merge steps that can be achieved in a common PR
- Add steps to wait for runtime payload builds
- Add peer pods steps

### CODE REVIEW

**Review by fitzthum — APPROVED** [review]
Nice. I will probably make a more aggressive PR later, but these are all good changes imo.

File: .github/ISSUE_TEMPLATE/release-check-list.md [hunk 1]
```diff
@@ -30,45 +30,43 @@ assignees: ''
...
-          * Change the version
+          Note that you can point to your own fork here, so you don't actually do changes in the other projects
+          before making sure this step works as expected.
+        * attestation-agent and td-shim
+          * https://github.com/kata-containers/kata-containers/blob/CCv0/versions.yaml
+            * Change the version
 
-- [ ] - 6. Update Kata Containers to use the latest td-shim
+- [ ] - 5. Wait for kata-runtime-payload-ci to be successfully built
+        * After the previous PR is merged wait for the kata-runtime-payload-ci (https://github.com/kata-containers/kata-containers/actions/workflows/cc-payload-after-push.yaml) has completed, so the latest kata-runtime-payload-ci containers the changes
```
Comment (wainersm) [inline .github/ISSUE_TEMPLATE/release-check-list.md:46]: s/containers/contains ? Yeah, it is containers everywhere all the time :)

File: .github/ISSUE_TEMPLATE/release-check-list.md [hunk 2]
```diff
@@ -30,45 +30,43 @@ assignees: ''
...
+- [ ] - 7. Ensure the Operator is using the latest CI builds and that the Operator tests are passsing
 
         * Enclave CC:
           * SIM: https://github.com/confidential-containers/operator/blob/main/config/samples/enclave-cc/sim/kustomization.yaml
           * HW: https://github.com/confidential-containers/operator/blob/main/config/samples/enclave-cc/base/ccruntime-enclave-cc.yaml
           * Note that we need the quay.io/confidential-containers/runtime-payload-ci registry and enclave-cc-{SIM,HW}-latest tags
         * Kata Containers:
-          * s390x: https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/s390x/kustomization.yaml
-          * x86_64: https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/default/kustomization.yaml
+          * default: https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/default/kustomization.yaml
```
Comment (wainersm) [inline .github/ISSUE_TEMPLATE/release-check-list.md:64]: Just for me to understand the recent changes: the runtime-payload is now multi-arch. The s390x overlay is only needed currently to restrict the runtimeClasses (i.e. makes no sense to have, say, kata-qemu-sev on s390x).

File: .github/ISSUE_TEMPLATE/release-check-list.md [hunk 3]
```diff
@@ -107,27 +105,41 @@ assignees: ''
...
 
+
+- [ ] - 21. Wait for release kata-runtime-payload to be successfully built
+        * After the Kata tag is created wait for (https://github.com/kata-containers/kata-containers/actions/workflows/cc-payload.yaml) to be successfully completed, so the latest commit kata-runtime-payload for the release is created
+
+- [ ] - 22. Contact @stevenhorsman and/or @bpradipt to get the peer pods release ready
+  - Update the versions of the peer pods dependencies:
+    - attestation-agent - set to the the release tag
+    - kata-containers - set to the release tag
+  - Create the new peer pods  release:
```
Comment (wainersm) [inline .github/ISSUE_TEMPLATE/release-check-list.md:122]: extract space pods\s\srelease

Comment (wainersm) [reply .github/ISSUE_TEMPLATE/release-check-list.md:122]: BTW, I just created an issue (https://github.com/confidential-containers/cloud-api-adaptor/issues/1108) suggesting that we have a release checklist on peer pods as well. We could have the content of this bullet detailed there, and then just link here.

File: .github/ISSUE_TEMPLATE/release-check-list.md [hunk 4]
```diff
@@ -107,27 +105,41 @@ assignees: ''
...
-- [ ] - 23. Cut an operator release
+- [ ] - 25. Cut an operator release
 
-- [ ] - 24. Make sure to update the release notes
+- [ ] - 26. Make sure to update the release notes
 
         * https://github.com/confidential-containers/documentation/tree/main/releases/v<TARGET_RELEASE>.md
         
-- [ ] - 25. Poke Jens Freimann (jfreiman@redhat.com) to update the release to the OperatorHub
+- [ ] - 27. Poke Jens Freimann (jfreiman@redhat.com) to update the release to the OperatorHub
```
Comment (wainersm) [inline .github/ISSUE_TEMPLATE/release-check-list.md:145]: I spoken with @jensfr  and I will take over this task.

File: .github/ISSUE_TEMPLATE/release-check-list.md (hunk 1 above)
Comment (stevenhorsman) [reply .github/ISSUE_TEMPLATE/release-check-list.md:46]: Thanks for spotting this - at least I didn't autocomplete to containerd that I have a habit of doing 😿

File: .github/ISSUE_TEMPLATE/release-check-list.md (hunk 2 above)
Comment (stevenhorsman) [reply .github/ISSUE_TEMPLATE/release-check-list.md:64]: Yeah - that was a change I introduced a few weeks ago in https://github.com/confidential-containers/operator/pull/169/commits/44ff27b35bf3917145057afc85810e97b7e72d97

**Comment by stevenhorsman** [comment]
@wainersm - thanks for the review and sorry for my poor english. I've kept the updates as a separate commit for ease of re-review, but if/when you are happy let me know and I'll fix-up them.

**Comment by wainersm** [comment]
> @wainersm - thanks for the review and sorry for my poor english. I've kept the updates as a separate commit for ease of re-review, but if/when you are happy let me know and I'll fix-up them.

ywc @stevenhorsman ! My perfect english skills I put in your service ;)

**Review by wainersm — APPROVED** [review]
Thanks @stevenhorsman !

File: .github/ISSUE_TEMPLATE/release-check-list.md (hunk 3 above)
Comment (stevenhorsman) [reply .github/ISSUE_TEMPLATE/release-check-list.md]: I'm going to suggest waiting for a day or so to merge this and I'll try and move/write the release process in the cloud-api-adaptor and then link to that as suggested.

Comment (wainersm) [reply .github/ISSUE_TEMPLATE/release-check-list.md]: Works for me.

Comment (stevenhorsman) [reply .github/ISSUE_TEMPLATE/release-check-list.md]: I've created https://github.com/confidential-containers/cloud-api-adaptor/pull/1110 for this, so once it gets reviewed and merged, we can link to sections in there.

**Pull request merged**